docs/bugs.json

```diff
@@ -1,4 +1,17 @@
 [
+    {
+        "name": "ABIEncoderV2LoopYulOptimizer",
+        "summary": "If both the experimental ABIEncoderV2 and the experimental Yul optimizer are activated, one component of the Yul optimizer may re-use data in memory that has been changed in the meantime.",
+        "description": "The Yul optimizer incorrectly replaces ``mload`` and ``sload`` calls if there was a matching ``mstore`` or ``sstore`` call before and the memory or storage was only changed in a function that was called in between the store and the load call and that function has to contain a loop where the same memory location is changed in the loop body. When used in Solidity mode, this can only happen if the experimental ABIEncoderV2 is activated and the experimental Yul optimizer has been activated manually in addition to the regular optimizer in the compiler settings.",
```

Comment:
```suggestion
        "description": "The Yul optimizer incorrectly replaces ``mload`` and ``sload`` calls with values that have been previously written to the load location if all of the following conditions are met: (1) there is a matching ``mstore`` or ``sstore`` call before; (2) the contents of memory or storage is only changed in a function that is called in between the first store and the load call; (3) the called function contains a for loop where the same memory location is changed in the init, post, or body blocks. When used in Solidity mode, this can only happen if the experimental ABIEncoderV2 is activated and the experimental Yul optimizer has been activated manually in addition to the regular optimizer in the compiler settings.",
```

Reply:
Split the large sentence into smaller chunks for ease of understanding. Other than that, changed `store must happen in loop body to store must happen in for loop init, post, or body blocks`